src/scenario.jl

```diff
@@ -646,7 +660,7 @@ function run_model(domain::Domain, param_set::YAXArray, functional_groups::Vecto
 
     # Cache matrix to store potential settlers
     potential_settlers = zeros(size(fec_scope)...)
-    linear_extension = _to_group_size(domain.coral_growth, corals.linear_extension)
+    _linear_extensions = linear_extensions() # _to_group_size(domain.coral_growth, corals.linear_extension)
```

Comment:
This breaks linear extension values being provided by the sampler.

Reply:
Ok, I'll fix that.